etna/ciim/client.py

```diff
@@ -296,6 +300,37 @@ def fetch(
             raise MultipleObjectsReturned
         return result_list.hits[0]
 
+    def fetch_iiif_manifest(
+        self,
+        *,
+        id: str,
+    ) -> IIIFManifest:
+        """Make request and return response for Client API's IIIF manifest endpoint.
+
+        Used to fetch a single item by its identifier.
+
+        Keyword arguments:
+
+        id:
+            Matches a single IAID.
+            Ex: returns match on Information Asset Identifier - iaid.
+        """
+        response = self.make_request(self.get_public_iiif_manifest_url(id=id))
```

Comment:
**Question:** Is there a need to implement a fake request of some kind for the CI tests to not hit the real server?

Reply:
I think we have some `records.json`s that we use for this - I haven't been too involved in the "search" applications though, so not sure what state we're in at the minute. If you want to make some tests that you feel are appropriate, feel free! 😃

Reply:
Mhm... I am hesitant to add tests I can't run with confidence because the existing unit tests are not ready for Rosetta. I don't want to silently introduce broken tests. I'll push the testing to #1554.